spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -169,6 +169,46 @@ cf start dataflow-server
 
 Alternatively, you can run the Admin application locally on your machine which is described in the next section.
 
+[[sample-manifest-teample]]
+==== Sample Manifest Template
+
+As an alternative to setting environment variables via `cf set-env` command, you can curate all the relevant env-var's
+in `manifest.yml` file and use `cf push` command to provision the server.
+
+Following is a sample template to provision the server on PCFDev.
+
+[source,bash,subs=attributes]
+----
+---
+applications:
+- name: <PREFERRED NAME OF THE SERVER APP>
+  host: <PREFERRED HOST>
+  memory: <PREFERRED MEMORY>
+  disk_quota: <PREFERRED DISKQUOTA>
+  timeout: <PREFERRED API TIMEOUT>
+  instances: <NO OF INSTANCES>
+  path: <ABSOLUTE PATH TO SERVER UBER-JAR> # (eg., /demo/spring-cloud-dataflow-server-cloudfoundry-1.1.0.RELEASE.jar)
+  env:
+    SPRING_APPLICATION_NAME: <PREFERRED NAME OF THE SERVER APP>
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_URL: https://api.local.pcfdev.io # (eg., URL for PCFDev)
```

Comment:
All those `#` comments don't really make sense to me. `eg.` doesn't mean "example value"